### PR TITLE
{bio}[foss/2019a,fosscuda/2019a] ctffind v4.1.13

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~#8125~, ~~#8141~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/cfc72861e0cc6cb44dffab480417fb86 for a full test report.

**Comment by vanzod** [comment]
Re-trigger Travis

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/29d52b5bcf9cc187cc89c830e0bd277e for a full test report.

**Comment by vanzod** [comment]
@akesandgren In the previous easyconfig files we had a message regarding the initialization of `OMP_NUM_THREADS` or the use of `--omp-num-threads`. Does this not apply here?

**Comment by vanzod** [comment]
Moreover in this comment on a previous PR you thought about building it with `GCC` instead of `foss`/`fosscuda`. I would say it is the same here...
https://github.com/easybuilders/easybuild-easyconfigs/pull/8044#discussion_r274768084

**Comment by akesandgren** [comment]
The message about OMP_NUM_THREADS was just annoying, it applies to everything using OpenMP, so why have it only here? I just dropped it.

Reg using GCC instead, yes i forgot. I'll change this one to do so instead.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/60efd6db6be579183b44dce9d0da8447 for a full test report.

**Comment by akesandgren** [comment]
Gah.... it needs FFTW so must be at gompi(c) level.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/2e9b4cca07265bb268f253e91e6247e3 for a full test report.

**Comment by akesandgren** [comment]
Hmmm..... brain apparently not working well today...
it's much better to keep it at foss level so it pulls in the right FFT directly instead of having to specifically depend on FFT (or MKL) in the easyconfig.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/b8c42e4f045a151a7187125a8d8df88f for a full test report.

File: easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.2'),
+    ('LibTIFF', '4.0.10'),
+    ('GSL', '2.5'),
+    ('wxWidgets', '3.0.4'),
+]
+
+separate_build_dir = True
```
Comment (boegel) [inline easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-foss-2019a.eb]: @akesandgren This is meaningless when using `ConfigureMake`, only makes sense with `CMakeMake`?

File: easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-fosscuda-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.2'),
+    ('LibTIFF', '4.0.10'),
+    ('GSL', '2.5'),
+    ('wxWidgets', '3.0.4'),
+]
+
+separate_build_dir = True
```
Comment (boegel) [inline easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-fosscuda-2019a.eb]: meaningless here, should be dropped

**Comment by boegel** [comment]
Am I the only one wondering how the f*ck this can ever have worked without the patch?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3abbc856e541f1b73298013b0f9158d3 for a full test report.

File: easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-foss-2019a.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-foss-2019a.eb]: Yeah, but a correctly written autoconf setup can also use separate build dirs :-)

Dropping...

**Comment by akesandgren** [comment]
No you're not. I failed completely without the patch both by hand and in every other circumstance i tried.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/69588dde38b01766a36e91f1fc89257e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f8d3e31da25df4b57bcab0e1d18ea56b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2075.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c664601a3463285afa9286d88d07b81f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**